src/modules/core/components/ActionsList/ActionsList.tsx

```diff
@@ -26,7 +27,7 @@ const ActionsList = ({ items, handleItemClick }: Props) => (
   <ul className={styles.main}>
     {items.map((item) => (
       <ActionsListItem
-        key={item.id}
+        key={item.id || nanoid()}
```

Comment:
I still don't like this, since `nanoId` is a bit fiddly. 

First, change it to this:
```suggestion
        key={item.id || nanoid(`${item.name}${item.hash}`)}
```

Then, have a read here about using `nanoId` with react: https://www.npmjs.com/package/nanoid#react

Reply:
so.. nanoId have only optional number parameters - size
It wont take strings. Also i can't do just `${item.name}${item.hash}` because its not unique either 🤦‍♀️

Reply:
Ufff. Ok. 

There's another way of doing this using the `useState` hook, but it's pretty convoluted.

Leave just `nanoId()` for now so you can merge it.